src/components/App.tsx

```diff
@@ -3852,14 +3852,21 @@ class App extends React.Component<AppProps, AppState> {
         const dx = pointerCoords.x - draggingElement.x;
         const dy = pointerCoords.y - draggingElement.y;
 
-        const pressures = draggingElement.simulatePressure
-          ? draggingElement.pressures
-          : [...draggingElement.pressures, event.pressure];
+        const discardPoint =
+          points.length > 0 &&
+          points[points.length - 1][0] === dx &&
+          points[points.length - 1][1] === dy;
```

Comment:
```suggestion
        const lastPoint = points.length > 0 && points[points.length - 1];
        const discardPoint =
          lastPoint && lastPoint[0] === dx && lastPoint[1] === dy;
```

probably doesn't matter nowadays (and for this use case), but this makes me less nervous :)